deploy/olm-catalog/qdrouterd-operator/0.1.0/catalog-source.yaml

```diff
@@ -22,18 +21,28 @@ items:
             apiservicedefinitions: {}
             customresourcedefinitions:
               owned:
-              - version: v1alpha1
-                kind: Qdrouterd
-                description: An instance of Qdrouterd
+              - description: An instance of Qdrouterd
                 displayName: Qdrouterd
+                kind: Qdrouterd
```

Comment:
Please rearrange to avoid showing diff lines where there is no change

Reply:
changes made to fix those unnecessary write spaces and position movement.